docs/administration-guide/gaffer-deployment/gaffer-docker/gaffer-images.md

```diff
@@ -0,0 +1,115 @@
+# Gaffer Images
+
+As demonstrated in the [quickstart](../quickstart.md) it is very simple to start
+up a basic in memory gaffer graph using the available Open Container Initiative
+(OCI) images.
+
+For large scale graphs with persistent storage you will want to use a different
+storage backend to a Map Store; the recommended one being Accumulo. To do this a
+different deployment of containers are required. This guide will run through the
+containers needed for a basic Accumulo cluster and how to configure and create
+custom images of Gaffer.
+
+## Available Images
+
+Currently there are a few different images that can be used to run a Gaffer
+deployment. The main ones are outlined in the following table and are all
+available on [Docker Hub](https://hub.docker.com/u/gchq).
+
+| Image | Description |
+| ----- | ----------- |
+| `gchq/accumulo` | This image is a containerised deployment of [Apache Accumulo](https://accumulo.apache.org/). This was created as historically there has not been an available official image from the maintainers of Accumulo; however, there has since been an [offical image](https://github.com/apache/accumulo-docker) made available but it is not currently in use in Gaffer. |
+| `gchq/hdfs` | A custom image for running HDFS (Hadoop file system) via a container. Contains an official release of [Apache Hadoop](https://hadoop.apache.org/) which is used as the scalable data storage for Accumulo. |
+| `gchq/gaffer` | This is the main container image for Gaffer that is built on on top of the `gchq/accumulo` image so includes a release of `zookeeper`, `hdfs` and `accumulo` along with the Gaffer libraries. Running this image simply runs the Accumulo instance not a Gaffer instance. |
```

Comment:
This runs an instance of Accumulo with the Gaffer add-ons that let you create a graph in Accumulo.